x-pack/plugins/metrics_entities/kibana.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "metricsEntities",
+  "version": "8.0.0",
+  "kibanaVersion": "kibana",
+  "configPath": ["xpack", "metricsEntities"],
+  "server": true,
+  "ui": false,
+  "requiredPlugins": ["data", "dataEnhanced"],
```

Comment:
This is a great undertaking to bring use case specific transform capabilities to solutions!!

I just had a really brief look at the code, there might be some opportunities to reuse Kibana's `transform` plugin and its APIs. To make those API endpoints available, suggest to add the `transform` plugin here:

`"requiredPlugins": ["data", "dataEnhanced", "transform"],`

The API routes are defined here: https://github.com/elastic/kibana/tree/master/x-pack/plugins/transform/server/routes/api

The API integrations tests would give you an idea how to use them: https://github.com/elastic/kibana/tree/master/x-pack/test/api_integration/apis/transform

Happy to catch up or discuss if you have any more questions.

Reply:
Oh hey! Those are looking nice. Yeah, I will swap mine out for those. Thanks for the pointers.